### ISSUE TITLE
[BUG] Update the setup dialog's logo image

### ISSUE THREAD

**Issue by risanaka** [Microsoft/accessibility-insights-windows#292]
Please check whether the bug has [already been filed](https://github.com/Microsoft/accessibility-insights-windows/issues).

**Describe the bug**

When the user installs Accessibility Insights for Windows, Ada the cat is visible on the left side of the first install dialog. However, once you start the install dialog, there's a red box where the logo would be.

**To Reproduce**
1. Go to [Accessibility Insights](https://accessibilityinsights.io)
1. Select **Download for Windows**.
1. Run the downloaded MSI.
1. Select Next in the Setup dialog.

**Expected behavior**
There should be an Accessibility Insights logo at the top right.

**Actual behavior**
What actually happened.
There's a red square with a CD at the top right.

**Screenshots or .GIF**
![Setup dialog of Accessibility Insights for Windows showing the red square with a CD at the top right instead of a logo](https://user-images.githubusercontent.com/144273/56004087-57c9fa00-5c7e-11e9-9cd9-0417ee400c90.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10 1809] (Get the version by running `winver` from the command line) Windows 10 Version 1809
 - Accessibility Insights for Windows Version: latest available for download
 - Target Application: [e.g. Chrome, Wildlife Manager] not applicable
 - Target Application Version: [e.g. 22] not applicable

**Additional context**
Priority requested -

According to WiX, it looks like this is the [top banner](http://wixtoolset.org/documentation/manual/v3/wixui/wixui_customizations.html) and may be customizable (I haven't tried it)?

**Comment by ferBonnin** [comment]
@neocln thoughts on this?

**Comment by karanbirsingh** [comment]
@ferBonnin We are unable to reuse the initial-screen image because the dimensions are different. To replace the red CD image, we need anything of dimensions 493 × 58. Then we can add a new line referencing it with the variable WixUIBannerBmp like we do for the existing logo.

https://github.com/Microsoft/accessibility-insights-windows/blob/f315fbf501a71c0a635d5942b483f6b6a961cdda/src/MSI/Product.wxs#L24

**Comment by ferBonnin** [comment]
@cheade could you provide an image with those dimensions for the accessibility insights logo?

**Comment by cheade** [comment]
@karanbirsingh Hi! I'm having a bit of trouble understanding the 493 x 58 dimension. The original image shown is a square. Are the dimensions a typo, or does it allow for a banner-type image to replace the square?

**Comment by karanbirsingh** [comment]
@cheade It seems to me the square is actually right-aligned within a banner-type image. For example, here's what happens when I use our initial-screen image in the banner's place:

![image](https://user-images.githubusercontent.com/7775527/56326384-acf88680-612a-11e9-9d86-8026daedb8b1.png)

The dimensions come from [this link](http://wixtoolset.org/documentation/manual/v3/wixui/wixui_customizations.html), which show the possible changes we can make to the installer:

Variable name | Description | Dimensions
-- | -- | --
WixUIBannerBmp | Top banner | 493 × 58
WixUIDialogBmp | Background bitmap used on the welcome and completion dialogs | 493 × 312
WixUIExclamationIco | Exclamation icon on the WaitForCostingDlg | 32 × 32
WixUIInfoIco | Information icon on the cancel and error dialogs | 32 × 32
WixUINewIco | Button glyph on the BrowseDlg | 16 × 16
WixUIUpIco | Button glyph on the BrowseDlg | 16 × 16

**Comment by cheade** [comment]
@karanbirsingh thank you! That makes sense now. Let me get something designed for you and sent your way ASAP. I'll have a couple options to mull over.

**Comment by cheade** [comment]
@karanbirsingh @ferBonnin 
**Option 1**
![Option 1](https://user-images.githubusercontent.com/40274682/56327723-8c332f80-6130-11e9-83cc-7f2151bf1baf.png)

**Option 2**
![Option 2](https://user-images.githubusercontent.com/40274682/56327726-90f7e380-6130-11e9-8cfa-93e80d88f1b2.png)

I prefer "Option 1". Let me know your thoughts :)

**Comment by ferBonnin** [comment]
opt 1 feels cleaner to me.

**Comment by cheade** [comment]
![Desktop installation banner](https://user-images.githubusercontent.com/40274682/56329362-f3a0ad80-6137-11e9-84e7-d5353379c31e.png)

This banner image should work for you!

**Comment by karanbirsingh** [comment]
closing on risa's behalf
